docs/12-performance-test.md

```diff
@@ -55,32 +54,98 @@ the table below for the results for various versions of Sleeper.
 
```

Comment:
There's a bit here about using logs to get the performance values. That isn't relevant to the deployAll test any more, and can be done with the reports in the compactionPerformance test.